src/miragevpn_server.md

```diff
@@ -0,0 +1,125 @@
+The MirageVPN server is a unikernel which listens on a port for client
+connections, and provides Internet connectivity (with NAT) for all
+authenticated clients. A single network interface is used for both client
+connections and providing connectivity.
+
+The unikernel will encrypt all packets received from the Internet and send
+them to the respective client. All packets received from a client will be
+decrypted and forwarded to the Internet. If "client-to-client" is enabled
+in the configuration, packets from one client which destination is another
+client will be forwarded by the server.
+
+# Scope of the server
+
+The scope is at the moment limited to IPv4 (no IPv6), the server only listens
+on TCP (no UDP). Only layer 3 networking is supported (tun device), there's no
+support for tap devices.
```

Comment:
The *tunnel* traffic is IPv4 only. We could listen on an IPv6 address. The server would need a IPv4 address to NAT from I guess. I'm not sure this nuance is too interesting...

Reply:
done in 51a9dee